puffin/src/lib.rs

```diff
@@ -430,6 +476,17 @@ impl ThreadProfiler {
     }
 }
 
+/// Incremental monolithic counter to identify scopes.
+static SCOPE_ID_TRACKER: std::sync::atomic::AtomicU32 = std::sync::atomic::AtomicU32::new(1);
+
+fn fetch_add_scope_id() -> ScopeId {
+    let new_id = SCOPE_ID_TRACKER.fetch_add(1, std::sync::atomic::Ordering::SeqCst);
```

Comment:
```suggestion
    let new_id = SCOPE_ID_TRACKER.fetch_add(1, std::sync::atomic::Ordering::Relaxed);
```

Reply:
Its only called once I suppose so not that biggy, but yes can be relaxed